src/twister2/report/subtest_reporting_plugin.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import pytest
+from pytest_subtests import SubTestReport
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_report_teststatus(report, config):
+    """ This plugin modifies the one from SubTestReport.
+    We only want to see the subtests' status in the console if verbosity > 1."""
+
+    if report.when != 'call' or not isinstance(report, SubTestReport):
+        return
+
+    if hasattr(report, 'wasxfail'):
+        return None
+
+    # report.location[0] is used when verbosity >=2 by default plugin printing noise
+    # this change the useless path to testcase name
+    loc = list(report.location)
+    loc[0] = f'ztestcase "{report.context.msg}"'
+    report.location = tuple(loc)
+
+    output_formating = {
+        'passed': ['subtests passed', ',', 'SUBPASS'],
+        'skipped': ['subtests skipped', '-', 'SUBSKIP'],
+        'failed': ['subtests failed', 'f', 'SUBFAIL'],
+    }
+    # This removes the substatuses from console outputs if verbosity <= 1
+    if config.option.verbose <= 1:
+        for key in output_formating.keys():
+            output_formating[key][1] = ''
+            output_formating[key][2] = ''
+
```

Comment:
maybe would be simpler to add modification of report location for yaml files just here?
if config.option.verbose >= 2 and  '.yaml' not in report.fspath:
   ...
   report.location = tuple(loc)